full-service/src/json_rpc/mod.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+//! JSON-RPC-2.0 API
```

Comment:
Most of the code here isn't new. Most of the files here come from splitting decorated_types.rs into separate files.